### ISSUE TITLE
Question re: Build Error #![feature] may not be used on the stable release channel

### ISSUE THREAD

**Issue by joncarr** [Wilfred/remacs#1160]
I continually receive a build error `E0554` when trying to build.

I have read the comments regarding the same issue in issue#629 and have ensured that both super user and my user are running the same nightly toolchain.

The error reads:
```
Compiling remacs-macros v0.1.0 (/home/jec/code/rust/remacs/rust_src/remacs-macros)                                                                                                                                             
error[E0554]: #![feature] may not be used on the stable release channel                                                                                                                                                           
 --> remacs-macros/lib.rs:2:1                                                                                                                                                                                                     
  |                                                                                                                                                                                                                               
2 | #![feature(extern_crate_item_prelude)]                                                                                                                                                                                        
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                                                                                                        
                                                                                                                                                                                                                                  
error: aborting due to previous error                                                                                                                                                                                             
                                                                                                                                                                                                                                  
For more information about this error, try `rustc --explain E0554`.                                                                                                                                                               
error: Could not compile `remacs-macros`.                                                                                                                                                                                         

To learn more, run the command again with --verbose.
make[1]: *** [Makefile:614: ../rust_src/target/debug/libremacs_lib.a] Error 101
make[1]: Leaving directory '/home/jec/code/rust/remacs/src'
make: *** [Makefile:429: src] Error 2
```

Relevant information:

```
[root@jec-xxl remacs]# rustup show
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

nightly-2018-11-03-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu (default)

active toolchain
----------------

nightly-2018-11-03-x86_64-unknown-linux-gnu (overridden by '/home/jec/code/rust/remacs/rust-toolchain')
rustc 1.32.0-nightly (8b096314a 2018-11-02)

[root@jec-xxl remacs]# 
```

```
 jec@jec-xxl     ~/c/r/remacs     master ±  rustup show                                                                                                                                                          10:20 
Default host: x86_64-unknown-linux-gnu

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu
nightly-2018-11-03-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu (default)

active toolchain
----------------

nightly-2018-11-03-x86_64-unknown-linux-gnu (overridden by '/home/jec/code/rust/remacs/rust-toolchain')
rustc 1.32.0-nightly (8b096314a 2018-11-02)
```

Linux Distribution: ArcoLinux (Arch Based)
Architecture: 64-bit
gcc version: gcc (GCC) 8.2.1 20181127

I have used both users, ran `make clean` after each attempt.  Are there any additional steps I need to do in order to get this built?

**Comment by shaleh** [comment]
As the user that is seeing the error run `rustc --version` at the top of the remacs tree. Does it show stable or nightly?
Look in config.log. is the cargo and rust found what you expect?

**Comment by joncarr** [comment]
Running `rustc --version` showed that I was using stable, despite having the active toolchain set to nightly.  As it turns out, there was an old entry in my `.zshrc` that was exporting the path to the stable toolchain.  I must have set this when I was messing around with some emacs packages. Once I removed this I was able to build successfully, I apologize for the unnecessary chatter but I thank you for your time and pointing me in the right direction.
